tests/unit/dojson/fixtures/test_hep_record.xml

```diff
@@ -197,6 +197,21 @@
     <subfield code="o">ConfAcr</subfield>
     <subfield code="0">1</subfield>
   </datafield>
+  <datafield tag="773" ind1="1" ind2=" ">
+    <subfield code="c">R123</subfield>
+    <subfield code="n">2</subfield>
+    <subfield code="p">Phys.Rev.Lett.</subfield>
+    <subfield code="v">105</subfield>
+    <subfield code="y">2010</subfield>
+    <subfield code="m">note</subfield>
+    <subfield code="z">isbn</subfield>
+    <subfield code="x">pubinfo_freetext</subfield>
+    <subfield code="w">cnum</subfield>
+    <subfield code="t">confpaper_info</subfield>
+    <subfield code="r">reportNumber</subfield>
+    <subfield code="o">ConfAcr</subfield>
+    <subfield code="0">1</subfield>
+  </datafield>
```

Comment:
I mean this style: https://github.com/inspirehep/inspire-next/blob/dd8042ce0f1953e2e764b9c19ac4df196f5d5dce/tests/unit/dojson/test_dojson_conferences.py#L33-L50

A snippet with the MARCXML that you're considering, the expected result, two lines of boilerplate for running the code and the assertion. It's a style introduced by @bittirousku and @tomaszgy, and I think it's a pretty neat idea.